praw/const.py

```diff
@@ -1,6 +1,5 @@
 """PRAW constants."""
-
-from .endpoints import API_PATH  # noqa: F401
+from .endpoints import API_PATH  # noqa
```

Comment:
Why is F401 removed from here?

Reply:
I just deleted the path by accident and forgot to fully restore the original

Reply:
My IDE auto-optimized the import away and I tried to replace the line and didn't realize F401 was eplicitly stated